servers/display_server.h

```diff
@@ -318,6 +318,7 @@ class DisplayServer : public Object {
 	virtual Rect2i get_display_safe_area() const { return screen_get_usable_rect(); }
 
 	enum {
+		INVALID_SCREEN = -1,
```

Comment:
Should the value of `SCREEN_MAIN_WINDOW` be updated t differ from `INVALID_SCREEN`?

Reply:
`SCREEN_MAIN_WINDOW` is used only as input parameter and `INVALID_SCREEN` as output, so it does not matter if value is the same.